### PR TITLE
fsql 0.1.1 (new formula)

### PR DESCRIPTION
**Pull request by kashav**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/fsql.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/kshvmdn").mkpath
+    ln_s buildpath, buildpath/"src/github.com/kshvmdn/fsql"
+    system "go", "build", "-o", bin/"fsql"
+  end
+
+  test do
+    output = shell_output(bin/"fsql -version")
```
Comment (ilovezfs) [inline Formula/fsql.rb]: This needs to be a functional test.

**Review by ilovezfs — CHANGES_REQUESTED** [review]

File: Formula/fsql.rb (hunk 1 above)
Comment (kshvmdn) [inline Formula/fsql.rb]: Sorry, I think I'm misunderstanding; the tests seem to be passing now? Do you mean that they should be testing functionality?

**Comment by kshvmdn** [comment]
@ilovezfs Made the requested changes, hopefully it's good to go now. 😅

**Comment by ilovezfs** [comment]
@kshvmdn thanks. It would be good if the test actually did something like create a couple files and then issue a query that should only match one of them, or something like that.

**Comment by kshvmdn** [comment]
@ilovezfs `brew audit` is telling me to wrap in both `//` and `%r`, but tests are failing when I do that. Am I missing something here?

**Comment by ilovezfs** [comment]
@kshvmdn It'll request %r if there are slashes in the expression.

This would be a bit more idiomatic for brew:

```
  test do
    (testpath/"bar.txt").write("hello")
    (testpath/"foo/baz.txt").write("world")
    expected = <<-EOS.undent
      foo
      foo/baz.txt
    EOS
    assert_equal expected, shell_output("#{bin}/fsql SELECT name FROM foo")
    output = shell_output("#{bin}/fsql SELECT name FROM . WHERE name = bar.txt")
    assert_equal "bar.txt", output.chomp
    output = shell_output("#{bin}/fsql SELECT all FROM . WHERE size \\< 500gb")
    assert_equal "", output
  end
```

**Comment by kshvmdn** [comment]
@ilovezfs Cool; made those changes!

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
LGTM.

**Comment by ilovezfs** [comment]
@kshvmdn thanks for the new formula and your first contribution to Homebrew! :rocket:

**Pull request closed without merging**